apm-agent-plugins/apm-quartz-job-plugin/src/main/java/co/elastic/apm/agent/quartz/job/JobTransactionNameInstrumentation.java

```diff
@@ -56,6 +57,7 @@ public JobTransactionNameInstrumentation(ElasticApmTracer tracer) {
     @Override
     public ElementMatcher<? super TypeDescription> getTypeMatcher() {
         return isInAnyPackage(applicationPackages, ElementMatchers.<NamedElement>none())
+            .or(nameStartsWith("org.quartz.job"))
```

Comment:
This works, but I prefer the boolean expression more explicit and clearer for reading:
```suggestion
        (isInAnyPackage(applicationPackages, ElementMatchers.<NamedElement>none()).or(nameStartsWith("org.quartz.job")))
```
Not insisting on it though.